dev/README.md

```diff
@@ -0,0 +1,15 @@
+# Guidelines for formatting the SBSCL code
```

Comment:
Please change to "Guidelines for using code formatting configuration files" or something similar.